layouts/company/list.html

```diff
@@ -17,15 +17,32 @@
              Normalize company name for sorting by uppercasing and translating accented characters to equivalent Latin
              alphabet characters.
         */}}
-        {{ $clean_name := replaceRE "[ÅÁÀÂÄÃ]" "A" (upper .Params.name) }}
-        {{ $clean_name := replaceRE "[Ç]" "C" $clean_name }}
-        {{ $clean_name := replaceRE "[ÉÈÊË]" "E" $clean_name }}
-        {{ $clean_name := replaceRE "[ÌÍÎÏ]" "I" $clean_name }}
-        {{ $clean_name := replaceRE "[ÒÓÔÕÖØ]" "O" $clean_name }}
-        {{ $clean_name := replaceRE "[ŬÙÚÛÜ]" "U" $clean_name }}
-        {{ $clean_name := replaceRE "[Ñ]" "N" $clean_name }}
+        {{ $clean_name := replaceRE "[ÅÁÀÂÄÃÅÆΆ]" "A" (upper .Params.name) }}
+        {{ $clean_name := replaceRE "[B̌B́]" "B" $clean_name }}
+        {{ $clean_name := replaceRE "[ÇČĆ̣Ḉ]" "C" $clean_name }}
+        {{ $clean_name := replaceRE "[ĎÐD́]" "D" $clean_name }}
+        {{ $clean_name := replaceRE "[ÉÈÊËĚÊ̌ΕĖ́Ę́É̩Έ]" "E" $clean_name }}
+        {{ $clean_name := replaceRE "[F̌F́]" "F" $clean_name }}
+        {{ $clean_name := replaceRE "[ǦǴ]" "G" $clean_name }}
+        {{ $clean_name := replaceRE "[ȞH́Ή]" "H" $clean_name }}
+        {{ $clean_name := replaceRE "[ÌÍÎÏΊ]" "I" $clean_name }}
+        {{ $clean_name := replaceRE "[J̌J́]" "J" $clean_name }}
+        {{ $clean_name := replaceRE "[ǨḰ]" "K" $clean_name }}
+        {{ $clean_name := replaceRE "[Ľ]" "L" $clean_name }}
+        {{ $clean_name := replaceRE "[M̌Ḿ]" "M" $clean_name }}
+        {{ $clean_name := replaceRE "[ÑŃ]" "N" $clean_name }}
+        {{ $clean_name := replaceRE "[ÒÓÔÕÖØΌ]" "O" $clean_name }}
+        {{ $clean_name := replaceRE "[P̌Ṕ]" "P" $clean_name }}
+        {{ $clean_name := replaceRE "[Q̌Q́]" "Q" $clean_name }}
+        {{ $clean_name := replaceRE "[ŘŘ̩Ŕ]" "R" $clean_name }}
         {{ $clean_name := replaceRE "[ŚṠŜŠŞ]" "S" $clean_name }}
-        {{ $clean_name := replaceRE "[ÝŸ]" "Y" $clean_name }}
+        {{ $clean_name := replaceRE "[ŤT́]" "T" $clean_name }}
+        {{ $clean_name := replaceRE "[ŬÙÚÛÜ]" "U" $clean_name }}
+        {{ $clean_name := replaceRE "[V̌V́]" "V" $clean_name }}
+        {{ $clean_name := replaceRE "[W̌Ẃ]" "W" $clean_name }}
+        {{ $clean_name := replaceRE "[X̌X́Ȳ́]" "X" $clean_name }}
+        {{ $clean_name := replaceRE "[ÝŸΎ]" "Y" $clean_name }}
+        {{ $clean_name := replaceRE "[ŽŹ]" "Z" $clean_name }}
```

Comment:
Did you research for other variations as well? I think we are missing some chars from the greek alphabet.

Reply:
ohh I added most I could find, can you tell me which type of greek alphabet we are missing out?

Reply:
- Capital Beta: Β
- Capital Zeta: Ζ
- Capital Kappa: Κ
- ...

Check e.g. https://en.wikipedia.org/wiki/Greek_alphabet